### ISSUE TITLE
Node 8.3.0 compatibility

### ISSUE THREAD

**Issue by internalfx** [LearnersGuild/rethinkdb-changefeed-reconnect#7]
```
error rethinkdb-changefeed-reconnect@0.3.1: The engine "node" is incompatible with this module. Expected version "5.6.x".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

Won't install.

**Comment by GeoffreyPlitt** [comment]
Any update on this?

**Comment by internalfx** [comment]
@GeoffreyPlitt I've just been running my fork while waiting on this PR.

If you need to get going right away you can add this to your `package.json`

`"rethinkdb-changefeed-reconnect": "https://github.com/internalfx/rethinkdb-changefeed-reconnect.git",`

**Comment by blakek** [comment]
Just noting for others landing here that this should be closed.  It seems to have been resolved by #8.  We were about to skip this as a solution because this issue.
